.github/dependabot.yml

```diff
@@ -61,11 +61,11 @@ updates:
     directory: "/"
     schedule:
       interval: monthly
-    target-branch: "rel/0.12"
+    target-branch: "rel/0.13"
```

Comment:
What about the branch rel/0.12?

Reply:
We usually maintain two major branches. For now, they should be 0.13 and 0.14. I think 0.12 can be a big-fix branch only and it's not necessary to check its dependencies. 

How do you think?

Reply:
IMO, if we are still maintaining the rel/0.12 branch, we'd better check the dependencies for security and safety.

Reply:
Fixed